pypose/module/icp.py

```diff
@@ -0,0 +1,100 @@
+import torch
+from .. import lietensor
+from .. import knn, svdtf
+
+class ICP(torch.nn.Module):
+    r'''
+    This class implements the batched Iterative Closest Point (ICP) algorithm to find a
+    similarity transformation ( :math:`T` ) between two sets of 3-dimensional points
+    using Singular Value Decomposition (SVD). It's important to note that the solution
+    found is only a local optimum.
```

Comment:
Batched Iterative Closest Point (ICP) algorithm to find a rigid transformation between two sets of points using Singular Value Decomposition (SVD).

add warning section for 'It's important to note that the solution
    found is only a local optimum.' First sentence of the doc only explains its purpose.